python/ribasim/tests/test_model.py

```diff
@@ -155,7 +155,7 @@ def test_write_adds_fid_in_tables(basic, tmp_path):
     # for node an explicit index was provided
     nrow = len(model_orig.basin.node.df)
     assert model_orig.basin.node.df.index.name is None
-    assert model_orig.basin.node.df.index.equals(pd.Index(np.full(nrow, 0)))
+    # assert model_orig.basin.node.df.index.equals(pd.Index(np.full(nrow, 0)))
```

Comment:
@Hofer-Julian I needed to reset the index after filtering the table to make write work, but that fails this test. Shall we remove this one?

Reply:
Yes, let's remove it.